runatlantis.io/docs/access-credentials.md

```diff
@@ -57,6 +57,8 @@ GitHub App needs these permissions. These are automatically set when a GitHub ap
 
 ::: tip NOTE
 Since v0.19.7, a new permission for `Administration` has been added. If you have already created a GitHub app, updating Atlantis to v0.19.7 will not automatically add this permission, so you will need to set it manually.
+
+Since v0.22.3, a new permission for `Members` has been added, which is required for features that apply permissions to an organizations team members rather than individual users. Like the `Administration` permission above, updating Atlantis will not automatically add this permission, so if you wish to features that rely on checking team membership you will need to add this manually.
```

Comment:
> v0.22.3

This is the next patch release number, not sure if this will end up being the correct version.